falcon/media/multipart.py

```diff
@@ -582,6 +581,11 @@ class MultipartParseOptions:
     ``application/x-www-form-urlencoded`` media types.
     """
 
+    if TYPE_CHECKING:
+        _DEFAULT_HANDLERS: ClassVar[Handlers]
+    else:
+        _DEFAULT_HANDLERS = None
```

Comment:
Realized just now (not related to this PR): maybe it would be better to assign `self.media_handlers` (see below) to a copy of this classvar in `__init__`? Because now customizing these handlers modifies the default as well.

Reply:
I went on to file this separately as #2293.

Reply:
Makes sense